### PR TITLE
API-16012: Rescue failures for appeals healthcheck calls

### PR DESCRIPTION
**Pull request by hinzed1127**
## Description of change

In the event that health-check calls fail, we want to gracefully handle that and communicate that a service is down, rather than pass through a raw `5XX` error

## Original issue(s)
https://vajira.max.gov/browse/API-16012

## Things to know about this PR

It'd be nice to rescue a more specific exception, but given the direct path of the call, it seems like just rescuing `StandardError`s should be ok

### CODE REVIEW

**Review by jhwuz — APPROVED** [review]

**Review by rileyanderson — APPROVED** [review]

File: modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb [hunk 1]
```diff
@@ -44,6 +44,14 @@ def appeals_status_upstream_healthcheck
       health_checker = AppealsApi::HealthChecker.new
       time = Time.zone.now.to_formatted_s(:iso8601)
 
+      appeals_status = lambda {
+        begin
+          health_checker.appeals_services_are_healthy? ? 200 : 503
+        rescue
```
Comment (cilestin) [inline modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb:50]: Let's log the exception class & message that were rescued

File: modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb [hunk 2]
```diff
@@ -54,13 +62,21 @@ def appeals_status_upstream_healthcheck
...
     end
 
     def decision_reviews_upstream_healthcheck
       health_checker = AppealsApi::HealthChecker.new
       time = Time.zone.now.to_formatted_s(:iso8601)
 
+      decision_reviews_status = lambda {
+        begin
+          health_checker.decision_reviews_services_are_healthy? ? 200 : 503
+        rescue
```
Comment (cilestin) [inline modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb:75]: Log the exception & message here, too. We usually prefix the messages with something easily searchable like `AppealsApi` or `AppealsApi::Healthchecker`.

**Review by cilestin — CHANGES_REQUESTED** [review]
Lookin good, just needs some logging!

File: modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb (hunk 1 above)
Comment (hinzed1127) [reply modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb]: added 👍

File: modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb (hunk 2 above)
Comment (hinzed1127) [reply modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb]: added 👍

**Review by cilestin — APPROVED** [review]

**Pull request merged**